src/NHibernate/Dialect/Dialect.cs

```diff
@@ -2446,6 +2446,11 @@ public virtual string LowercaseFunction
 		/// </summary>
 		public virtual int MaxAliasLength => 18;
 
+		/// <summary>
+		/// The maximum number of parameters allowed in a query.
+		/// </summary>
+		public virtual int? MaxNumberOfParameters { get; } = null;
```

Comment:
Why not, similarly to `MsSql2000Dialect`, write it:
```c#
public virtual int? MaxNumberOfParameters => null;
```
?

Reply:
My bad, now is corrected.